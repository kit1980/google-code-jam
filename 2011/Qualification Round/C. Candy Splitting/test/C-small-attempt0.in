100
5
1 2 3 4 5
3
3 5 6
2
5 6
2
5 5
15
288762 332610 671654 557705 15582 35776 4500 937392 685906 523920 368274 451380 660014 573424 97047
15
474197 561258 494468 329622 764966 763300 815512 60913 566237 646351 249156 826886 330448 629295 389052
9
645951 402273 126585 187839 738705 712528 486341 431152 548338
6
792931 508253 628063 438621 435931 901488
3
24905 676402 668539
4
888474 549947 578737 867856
6
795667 588465 290275 73548 448904 488837
12
94469 269782 504233 758068 274029 541294 244544 252486 687344 123039 571693 418825
4
643432 339499 569484 282575
11
95075 438360 294477 654438 772459 543344 425620 977786 48676 374710 730279
15
7370 1418 6066 5099 3224 1220 1548 5397 424 3225 4118 1262 611 2671 9248
11
7138 1911 4620 2949 3872 7310 9246 6925 1932 8313 9916
15
8208 517 3131 2424 6726 7439 1617 4726 9814 2441 3800 7803 1231 6572 5671
11
7849 3842 4009 644 3119 1919 8158 9147 718 7297 912
6
210 6711 4662 1030 5587 6406
11
5650 9985 4970 7452 276 1798 241 2330 9263 2922 6461
7
1776 6234 296 3700 7894 7368 5096
13
4843 4533 3785 6445 7749 9986 2748 7327 9250 5695 5146 6747 1922
13
1285 1316 2433 9593 6305 3078 1126 5265 82 9368 5731 7459 1795
12
5026 4006 3326 7556 4883 4824 3726 5529 6409 8329 8500 2745
11
5974 8238 5106 499 6230 2213 5173 1505 3519 1820 12029
7
7436 5254 558 1843 5969 7298 6005
7
7544 1213 5371 4363 1460 3104 5537
15
9436 9373 7260 4839 2600 2799 3677 6352 8513 5212 8245 7226 9902 2867 15423
10
3472 2211 7472 8068 1695 8950 9145 6485 5796 9959
15
835 1841 7294 8871 4538 8117 747 9214 3879 956 5328 759 3260 8436 9029
4
8121 1814 5194 9568
13
421 2604 9048 3715 7470 2931 8530 4329 3836 8793 7439 1648 13934
10
6057 6738 2128 5112 5492 5030 7396 2028 7351 5950
12
184 1979 3743 2811 824 7607 732 2889 8183 5661 8755 9287
10
8592 5708 5828 4194 6146 9774 694 2385 1763 5363
8
5044 3574 2376 3447 607 4598 706 2838
8
9023 8284 8224 6516 2111 8313 6873 2216
13
4261 8727 9712 9104 5396 3745 8892 2271 5869 2890 8958 7598 10227
2
6710 4525
9
2262 3416 1416 123 635 1925 9866 5437 6348
11
9349 5981 3553 2571 1882 9471 3333 1099 9607 4122 7663
9
9114 9828 9137 2549 9294 911 5547 4427 3227
9
872 6465 7835 7718 38 5641 4367 6459 1167
12
8845 4502 5249 3907 1063 2195 8747 3062 1416 4347 8181 1846
6
4435 333 1251 6568 7049 6469
3
762 4628 4334
3
1241 7753 6800
6
7524 3579 6333 7423 8535 1891
14
9331 4961 2737 2979 9355 1779 6388 3828 2243 2702 1793 4178 8229 14147
14
6036 7564 6174 849 2530 6332 3368 4894 948 7186 2867 4860 4796 2794
8
6761 6388 2404 5222 4561 9134 6489 2847
13
2007 7300 1170 3981 4058 6317 6815 930 5245 4338 5088 3109 1356
11
880 8919 9972 9429 7561 6629 4599 4848 2428 3573 9047
9
8611 5247 4660 2629 2945 4561 4055 6519 8541
10
9551 5554 5953 7901 3823 2391 9459 8009 7094 7893
4
4784 7960 1831 272
13
6750 6074 1644 8569 4676 2250 4410 7435 1761 4029 7872 437 10855
11
6177 4673 8726 6505 8048 2758 4994 1940 3734 4533 6082
7
2846 4711 9249 8745 5757 6199 4411
3
7561 9473 14472
3
7932 4179 3759
9
6930 41 864 1050 4210 4281 2234 2193 7329
10
317 9325 1205 5270 6315 8637 5669 931 2019 7680
15
2118 4792 6980 4580 7431 5856 6391 3922 1215 260 1460 1618 6666 4620 583
4
1944 4181 4469 863
7
9883 631 3397 1915 920 8314 4489
4
4437 9229 7581 10437
13
960 1368 3894 478 7510 8673 7914 650 7198 4709 2821 6285 13652
5
2743 7522 4632 2147 5219
6
7562 7666 2391 1166 9773 4634
3
7401 8949 15900
10
1059 2444 1149 6243 6343 8634 1407 781 5565 4517
14
6240 9148 5659 9205 9540 3685 4609 5427 1245 2584 4704 1022 8728 8034
11
5726 7088 3208 447 5121 6353 7779 3269 5160 1601 3270
9
5442 6624 9900 558 2669 3661 632 934 11742
8
8885 5238 5169 7387 8983 3718 1017 4161
4
5855 8666 2978 8636
5
6195 7503 8750 7619 14993
6
2240 3801 3828 3291 4 1074
8
5847 1804 4157 5666 9532 2329 782 14575
15
3412 5388 5535 7103 9984 9006 4322 5059 2265 7436 161 93 2562 7564 5072
13
9541 9551 6175 7175 1992 6633 2544 2386 2110 9454 885 2167 3921
5
5924 6414 86 7360 3010
5
9132 214 4968 9471 5357
6
3999 4332 137 3489 5861 1214
3
5601 6043 634
15
1440 1877 1311 5068 2060 9529 5034 1298 4509 831 9906 9927 3808 7155 10351
7
868 1530 3664 3809 5366 2117 6812
7
9116 2381 389 396 8313 3397 9340
3
8439 5533 13674
6
2397 6015 432 3793 5538 1249
13
3529 8189 5424 6548 2465 2246 5802 7054 585 1552 2262 8397 16017
5
5857 9333 183 5732 9287
11
3299 3880 4674 4702 8251 1541 9959 2065 2677 798 1986
7
3292 9757 303 3467 1849 7423 15779
10
8245 2284 504 302 4842 5226 9848 5496 4279 2478
6
6657 549 8230 5855 8697 3876
5
8689 4610 8225 1814 1931
11
7884 9224 5734 309 2632 6589 7084 9241 8084 9848 6067
2
491 1197
