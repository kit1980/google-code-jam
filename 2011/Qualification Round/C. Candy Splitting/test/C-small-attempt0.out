Case #1: NO
Case #2: 11
Case #3: NO
Case #4: 5
Case #5: 6199446
Case #6: NO
Case #7: NO
Case #8: NO
Case #9: 1344941
Case #10: 2335067
Case #11: 2612148
Case #12: 4645337
Case #13: 1552415
Case #14: NO
Case #15: NO
Case #16: NO
Case #17: 71603
Case #18: NO
Case #19: 24396
Case #20: NO
Case #21: 32068
Case #22: 74254
Case #23: 54754
Case #24: NO
Case #25: 51807
Case #26: NO
Case #27: 27379
Case #28: 101124
Case #29: NO
Case #30: 72357
Case #31: NO
Case #32: 74277
Case #33: 51254
Case #34: NO
Case #35: NO
Case #36: 22583
Case #37: 49449
Case #38: 85379
Case #39: NO
Case #40: NO
Case #41: NO
Case #42: 53123
Case #43: 40524
Case #44: 52297
Case #45: NO
Case #46: 8962
Case #47: 14553
Case #48: NO
Case #49: 72871
Case #50: 60349
Case #51: NO
Case #52: 50784
Case #53: NO
Case #54: 45139
Case #55: 65237
Case #56: NO
Case #57: 66325
Case #58: NO
Case #59: 39072
Case #60: 23945
Case #61: 12111
Case #62: 29091
Case #63: 47051
Case #64: 58232
Case #65: NO
Case #66: NO
Case #67: 27247
Case #68: 65634
Case #69: NO
Case #70: NO
Case #71: 24849
Case #72: NO
Case #73: 78808
Case #74: 48575
Case #75: 41604
Case #76: 43541
Case #77: NO
Case #78: 38865
Case #79: 14234
Case #80: 43910
Case #81: 74869
Case #82: NO
Case #83: NO
Case #84: 28928
Case #85: 18895
Case #86: 11644
Case #87: 73273
Case #88: 23298
Case #89: NO
Case #90: 22113
Case #91: 18992
Case #92: 69485
Case #93: 30209
Case #94: NO
Case #95: 41567
Case #96: NO
Case #97: 33315
Case #98: NO
Case #99: NO
Case #100: NO
